views/default/page/elements/comments.php

```diff
@@ -19,36 +19,29 @@
 	$limit = elgg_trigger_plugin_hook('config', 'comments_per_page', [], 25);
 }
 
-$id = '';
-if (isset($vars['id'])) {
-	$id = "id=\"{$vars['id']}\"";
-}
-
-$class = 'elgg-comments';
-if (isset($vars['class'])) {
-	$class = "$class {$vars['class']}";
-}
+$attr = [
+	'id' => elgg_extract('id', $vars, 'comments'),
+	'class' => (array) elgg_extract('class', $vars, []),
+];
+$attr['class'][] = 'elgg-comments';
```

Comment:
It's true that `[class="a b"]` would not match `<div class="b a">` but seems not worth worrying about.